### PR TITLE
Adding Visualizations Folder

### PR DESCRIPTION
**Pull request by Jace-Hambrick**
Adding folder for visualization documents/files.

Adding brainstorm_visuals.py to the Visualization folder.
contains information on Data Ethics and rough visualization ideas.

### CODE REVIEW

File: visualizations/visuals.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+#              Again. HRFA isn't there to put people at risk or bring them harm. However, it is possible
+#              that by putting their information on site it may cause them complications. Don't use pictures,
+#              FIRST and LAST names in conjunction, DOB information, as this may lead to legal complications for HRFA.
+#
+#   d) avoid using deceptive practices
+#
+#   e) give participants the right to withdraw from your research
+#              Note: bring this up to Shareholders. It's possible to gain animosity by not allowing someone to withdraw
+#              the information you are using about them. 
+#
```
Comment (boscolio) [inline visualizations/visuals.py]: These are all fantastic points that should be addressed with the stakeholder before live deployment.

**Comment by boscolio** [comment]
Great additions to consider when dealing with legal documents.  It will be important to get guidance from the stakeholder.

**Review by boscolio — COMMENTED** [review]
Great concepts to consider.  When dealing with legal cases there are a lot of boundaries we must be careful not to overstep.

**Review by begogineni — APPROVED** [review]
- Love the multiple points in the Data Ethics Things to Know section. Very in-depth
- Visualization softwares are looking good. 
- Great ideas for graphs - Only note to change is that we won't need a World Map. Only a map of the US.

**Review by boscolio — APPROVED** [review]

**Pull request merged**